### DISCUSSION TITLE
Translation Overlay

### DISCUSSION THREAD

**Discussion by Infinito21** [SethRobinson/UGT#55] [General]
Hello Robinson, I recently had the same problem with games in another language, I had a development idea and while looking for information I happened to discover your project and what you achieved seems great to me, I was reading the development and I am interested in knowing how you managed to make it translation will be positioned just above the source text, if you would tell me the class where you implemented the above I would appreciate it, regards.

**Comment by SethRobinson** [comment]
Hi, there isn't really a single class as such, it's just that the entire app is using raw win32 windows commands to create and control the window.  Win32 commands include: CreateWindowEx, SetWindowPos, SetForegroundWindow, etc.  This allows the app to be more flexible and do weird things that say, a game engine like Unity wouldn't allow.
